gnocchi/cli.py

```diff
@@ -190,36 +190,21 @@ class MetricProcessor(MetricProcessBase):
     def __init__(self, worker_id, conf):
         super(MetricProcessor, self).__init__(
             worker_id, conf, conf.metricd.metric_processing_delay)
-        self.coord, __ = utils.get_coordinator_and_start(
+        self.coord = utils.get_coordinator_and_start(
             conf.storage.coordination_url)
         self._tasks = []
         self.group_state = None
 
     @utils.retry
     def _configure(self):
-        self.store = storage.get_driver(self.conf, self.coord)
```

Comment:
oh now, i know why. i did this to reuse the coordinator for storage. i think we still want that?

 c307faa535a00b5525b73bb29a434ba2692f8c67